### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'split'

### ISSUE THREAD

**Issue by Brainiarc7** [Eyevinn/hls-to-dash#7]
Hello there,

Using this tool to generate Multi period MPEG DASH fails with the error above:

`hls-to-dash /path-to-master-playlist/master.m3u8 --multi > /outdir/mpd/stream.mpd `

Output:

```
Traceback (most recent call last):
  File "/usr/local/bin/hls-to-dash", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/hls2dash/__init__.py", line 36, in main
    mpd.load()
  File "/usr/local/lib/python2.7/dist-packages/hls2dash/lib/MPD.py", line 231, in load
    self._parseMaster(m3u8_obj)
  File "/usr/local/lib/python2.7/dist-packages/hls2dash/lib/MPD.py", line 412, in _parseMaster
    (video_codec, audio_codec) = stream.codecs.split(',')
AttributeError: 'NoneType' object has no attribute 'split'
```

**Comment by birme** [comment]
Thank you for the report. Do you have the ability to attach or link to the master-manifest that fails and we can easier reproduce it

**Comment by Brainiarc7** [comment]
Yes, sure.

**Comment by Brainiarc7** [comment]
An example of such a snippet can be generated by:

`ffmpeg -f lavfi -i color=red -f lavfi -i anullsrc -g 29.97 -r 29.97 -b:v:0 1000k -b:v:1 256k -b:a:1 64k -b:a:1 32k  -map 0:v -map 1:a -map 0:v -map 1:a -f hls -var_stream_map "v:0,a:0  v:1,a:1" -master_pl_name  master.m3u8 -t 360 -hls_time 2 /web/cdn/b%v/output_sl.m3u8`

Change the output path. Your master playlist as generated by FFmpeg will be at a top-level directory relative to the subdirectories with the variant HLS segments (b0,b1 and b2), named `master.m3u8`.

**Comment by Brainiarc7** [comment]
Hello,

This issue is fixed by applying the patch highlighted in #6 , [here](https://github.com/Eyevinn/hls-to-dash/pull/6).

**Comment by Brainiarc7** [comment]
Marking as closed.
